frontend/src/libs/components/table/table.tsx

```diff
@@ -56,13 +59,28 @@ const Table = <T,>({
       {table.getHeaderGroups().map((headerGroup) => (
         <tr key={headerGroup.id} className={styles.tr}>
           {headerGroup.headers.map((header) => (
-            <th key={header.id} className={styles.th}>
+            <th
+              key={header.id}
+              className={styles.th}
+              style={{
+                width: header.getSize(),
+              }}
+            >
               {header.isPlaceholder
                 ? null
                 : flexRender(
                     header.column.columnDef.header,
                     header.getContext(),
                   )}
+              <div
+                {...{
+                  onMouseDown: header.getResizeHandler(),
+                  onTouchStart: header.getResizeHandler(),
+                  className: `${styles.resizer} ${
+                    header.column.getIsResizing() ? `${styles.isResizing}` : ''
+                  }`,
```

Comment:
Maybe use getValidClassNames here?
And maybe use `&&` instead of the ternary operator?
```suggestion
                  className: `${styles.resizer} 
                  ${header.column.getIsResizing() && `${styles.isResizing}`}`,
```